infra/core/host/appservice.bicep

```diff
@@ -0,0 +1,113 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+// Reference Properties
+param applicationInsightsName string = ''
+param appServicePlanId string
+param keyVaultName string = ''
+param managedIdentity bool = !empty(keyVaultName)
+param AzureCognitiveSearch string
```

Comment:
These settings shouldn't be in core, but in /app.

Please keep consistent casing. Camel for params.